jenkins/swift-functional-tests/extras.d/55-swift-sproxyd.sh

```diff
@@ -23,6 +23,35 @@ function enable_sproxyd_driver {
     done
 }
 
+function amend_swift_conf {
+    local swift_conf=${SWIFT_CONF_DIR}/swift.conf
+    iniset ${swift_conf} storage-policy:1 name Policy-1
+}
+
+function create_storage_policies_conf {
+    cat <<EOF >${SWIFT_CONF_DIR}/scality-storage-policies.ini
+[ring:paris-arc]
+location = paris
+sproxyd_endpoints = http://127.0.0.1:81/proxy/arc
+
+[storage-policy:1]
+read = paris-arc
+write = paris-arc
+
+EOF
+}
+
+function sylimk_ring_files {
+    ln -s ${SWIFT_CONF_DIR}/object.ring.gz ${SWIFT_CONF_DIR}/object-1.ring.gz
+    ln -s ${SWIFT_CONF_DIR}/container.ring.gz ${SWIFT_CONF_DIR}/container-1.ring.gz
```

Comment:
Why do you have to create another container ring ? 

This documentation here : http://docs.openstack.org/developer/swift/development_saio.html (take a look at the remakerings script) create one additional objects ring for each new policy. But there's one/1 container ring in total.